components/usage-api/usage/v1/usage.proto

```diff
@@ -34,10 +34,18 @@ message ListBilledUsageRequest {
     }
 
     Ordering order = 4;
+
+    int64 per_page = 5;
+    int64 page = 6;
```

Comment:
Encapsulate the pagination in a message.

Reply:
👍🏻 thanks for the hint to encapsulate the pagination concerns.

Reply:
though this is nice for the reuse, it's really ugly on call site.
I'm not arguing agains that for the sake of the timeline, so let's do this.